html/details.html

```diff
@@ -0,0 +1,88 @@
+<html>
```

Comment:
the !DOCTYPE html tag should be added
https://developer.mozilla.org/en-US/docs/Glossary/Doctype
https://stackoverflow.com/a/6076465